client/testFixture/diagnostics.txt

```diff
@@ -0,0 +1 @@
+ANY browsers, ANY OS.
```

Comment:
What do we use this file for? If nothing, please remove.

Reply:
Same thing for the diagnostics.omt, diagnostics.odt, completion.txt, and completion.omt files

Reply:
they were used for tests of example functionality of the language server. will remove